tests/testthat/test-generate_data.R

```diff
@@ -12,3 +12,17 @@ test_that("generate_data works as expected", {
   expect_identical(result$id, 1:10)
   expect_integer(result$type_of_event, lower = 0, upper = 3, any.missing = FALSE)
 })
+
+test_that("show bug in v1 of generate_data", {
```

Comment:
```suggestion
test_that("generate_data correctly ... ", {
```
and in the `...` part describe what it is doing correctly / in which special case